### ISSUE TITLE
--name=never removes lines

### ISSUE THREAD

**Issue by nkh** [oniony/TMSU#260]
$ tmsu tags has_tags has_no_tags has_tags
has_tags: some_tag
has_no_tags:
has_tags: some_tag

3 lines are printed

$ tmsu tags --name=never has_tags has_no_tags has_tags
some_tag
some_tag

2 lines are printed!

**Comment by oniony** [comment]
Yeah, it currently prints the tags for each file. But you're right it would
be better if it printed the entire set of tags for all files when no name
is specified

**Comment by nkh** [comment]
@oniony 

It does print all the tags when no name is specified!

what it doesn't is print an empty line for the files that have no tags which means that with --name=never one doesn't know which file has what tags.

**Comment by oniony** [comment]
So I guess it should always print the name when there is more than one file.
